codecov.yml

```diff
@@ -6,7 +6,7 @@ coverage:
     project:
       default:
         # basic
-        target: 90%
+        target: 50%
```

Comment:
why 50%?

Reply:
I think this was a holdover from a previous iteration, just updated it!